mods/ts/rules/aircraft.yaml

```diff
@@ -238,6 +241,7 @@ SCRIN:
 		ROT: 3
 		Speed: 168
 		AirborneUpgrades: airborne
+		MoveIntoShroud: false
```

Comment:
iirc all air units handled pretty much the same except for attack behavior. They all landed when idle. I also noticed that with vision that low and fog of war the banshees attack move is nearly useless, since he can't turn fast enough to actually attack any unit after noticing it